### ISSUE TITLE
Large World ChunkSize is too big.

### ISSUE THREAD

**Issue by schmidt-florian** [FlaxEngine/FlaxEngine#1854]
**Issue description:**
If you create a large world, it works partially.

Large Worlds OFF: 

![image](https://github.com/FlaxEngine/FlaxEngine/assets/14238126/57358316-55e7-4c1b-9705-1fda17dfe46d)

Large Worlds ON:

![image](https://github.com/FlaxEngine/FlaxEngine/assets/14238126/e4f51f2d-b6ce-4a81-87e5-225c4ca4b356)

As you can see, large worlds works: BUT: The camera rotation needs a small distance to the world origin. But the camera rotates super clunky. That is because the Chunksize is 262144.

I set it for a test to just 4096 and it worked perfectly smooth even when the camera was at 1000000000 position.

So my question is .. why 262144 ? Has this any reason? Or could we set it to something way lower? So it is in the floating point precision loss. The best would be under 10k. Maybe 4096 or 8192. If this does not hurt.

**Steps to reproduce:**

1. Open the attached project with the large world engine compiled and open the scene "Large Worlds"
2. Press play
3. Downright in Game UI: Make sure Large World is checked
4. And enter a large number in the field like 10000000
5. Press Goto // This will teleport the setup to the given X Position from the TextBox.
6. The camera rotates always slightly and you will see a jittering 
7. If you recompile the engine with `Source\Engine\Level\LargeWorlds.h:22`set to 4096 (for example)
8. Repeat everything it will rotate smoothly as at would be at position 0 even it is at position 10000000.

**Minimal reproduction project:**

[LargeWorldBug.zip](https://github.com/FlaxEngine/FlaxEngine/files/13229231/LargeWorldBug.zip)

**Flax version:**
1.7 Large Worlds

**Comment by mafiesto4** [comment]
Yeah let's make it smaller. `8192` seams reasonable and better than the current value.

**Comment by NoriteSC** [comment]
> Yeah let's make it smaller. `8192` seams reasonable and better than the current value.

the chunk size can be 20,000 units from orgin without any problems 
but going past the 20k will start to be choppy for any actor with is rotating

**Comment by schmidt-florian** [comment]
It has no performance impact whether the ChunkSize is larger or smaller. It is only used to snap a position to an Origin. So there are only disadvantages to going higher and higher.

https://github.com/FlaxEngine/FlaxEngine/blob/c0a8d29453d1388c5f71c7814e9a3d5af5d742a9/Source/Engine/Level/Level.cpp#L51-L59

It's better to stay a little smaller. While you don't notice anything about the camera rotation etc. at 20k, Z-fighting can already occur.

For example, this Large World bug was only discovered after someone in Discord said they were having problems starting their FPS weapon Z-fighting at around 20k. Large Worlds was then recommended, but it didn't help. That's why I tested it and found this bug. That's why I suggested staying under 10k.
Even if the largest float error at 20k is "only" 0.0009.
But of course, you are right, for rotation, it would not matter.
